### ISSUE TITLE
Lip Upper A and Lip Lower A are not editable

### ISSUE THREAD

**Issue by salmoooon** [XIV-Tools/CustomizePlus#193]
Position, rotation, and scale will not work on both Lip Upper A and Lip Lower A.

Race: Lalafell male

Please let me know if more information is required!

**Comment by StoiaCode** [comment]
Yeah those bones are just kinda fucky wucky. They behave different to all others internally for some reason. Can check someday but it's admittedly very low prio.
